.github/workflows/update-version.yml

```
@@ -0,0 +1,30 @@
+    name: Update Package Version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+        with:
+          repository: gsainfoteam/icarus-gitops
+          ssh-key: ${{ secrets.SSH_PRIVATE_KEY }}
+          fetch-depth: 0
+
+      - name: Update Package Version
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2006:style:1:9: Use $(...) notation instead of legacy backticks `...`

(shellcheck)